consensus/consensus_fbft.go

```diff
@@ -14,14 +11,10 @@ func NewLockedFBFTPhase(initialPhrase FBFTPhase) *LockedFBFTPhase {
 }
 
 func (a *LockedFBFTPhase) Set(phrase FBFTPhase) {
```

Comment:
I think the parameter name should be  **phase** , please spell check

Reply:
Removed